### PR TITLE
Revert "Update cc1b1c03798884c636703443a23b8d9e827d6c0417921600394198a0.md"

### PR DESCRIPTION
**Pull request by AstroWa3l**
Reverts armada-alliance/armada-alliance#126

### CODE REVIEW

**Comment by AstroWa3l** [comment]
@sayshar need to fix your template header, the place where you defined the nodes.

File: services/website/content/en/stake-pools/cc1b1c03798884c636703443a23b8d9e827d6c0417921600394198a0.md [hunk 1]
```diff
@@ -5,22 +5,4 @@ memberSince: 2021-05-01
 identities: 
     - id: sayshar-srn
       role: spo
-nodes
```
Comment (AstroWa3l) [inline services/website/content/en/stake-pools/cc1b1c03798884c636703443a23b8d9e827d6c0417921600394198a0.md:8]: nodes:

File: services/website/content/en/stake-pools/cc1b1c03798884c636703443a23b8d9e827d6c0417921600394198a0.md [hunk 2]
```diff
@@ -5,22 +5,4 @@ memberSince: 2021-05-01
 identities:
```
Comment (AstroWa3l) [inline services/website/content/en/stake-pools/cc1b1c03798884c636703443a23b8d9e827d6c0417921600394198a0.md:5]: indendentaion?

**Comment by sayshar** [comment]
I submitted a new PR that should resolve the issue. Please see if that fixes it. I have double checked and added the missing colon which seems to have resolved it. You can close this PR and merge the new PR I suppose to avoid conflict.

**Pull request closed without merging**